### DISCUSSION TITLE
非激活标签可关闭

### DISCUSSION THREAD

**Discussion by sonichy** [linuxdeepin/developer-center#2851] [Features Request & Ideas | 特性请求 & 头脑风暴]
对比各种多标签软件，非激活标签都可以关闭。
deepin-editor 非激活标签关闭按钮是隐藏的，关闭不了。

**Comment by guoshaoyu** [comment]
> 对比各种多标签软件，非激活标签都可以关闭。
> deepin-editor 非激活标签关闭按钮是隐藏的，关闭不了。

感谢您的建议，我们会合理考虑此功能。
